modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/HectorBasedAndesContextStoreImpl.java

```
@@ -672,6 +672,13 @@ public void close() {
     }
 
     /**
+     * {@inheritDoc}
+     */
+    @Override public void clearSlotStorage() throws AndesException {
+        throw new NotImplementedException();
+    }
+
```

Shall we throw an UnsupportedOperationException?